### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by CandidDisk** [esin/esin#2] [Guestbook]
How come you follow so many people?

**Comment by esin** [comment]
Look at repo https://github.com/esin/ghfollow
Small bot, which follows to who followed to somebody

**Comment by hamiecod** [comment]
oh! the bot is pretty cool ;)

**Comment by lakshaysethi** [comment]
Hi! I saw your github contributions last of last year what were you working on? 200+ contributions a day! wow

**Reply by esin** [reply]
Yep
A - automatization 😄
